lessons/beginners/install-editor/info.yml

```diff
@@ -1,10 +1,12 @@
 title: Instalace editoru
 style: md
-attribution: Pro PyLadies Brno napsal Petr Viktorin, 2014-2017
+attribution: Pro PyLadies Brno napsal Petr Viktorin, 2014-2017, Tomáš Roj (2019)
 license: cc-by-sa-40
 subpages:
```

Comment:
@encukou Does the order here matter? If so, we should probably make vscode first.

Reply:
Right now the order isn't used anywhere, but it does make sense to put vscode first.